diff --git a/CMakeLists.txt b/CMakeLists.txt
index 255c305..b3c1559 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -47,6 +47,9 @@ if(NOT CMAKE_BUILD_TYPE)
   set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug" "Release" "MinSizeRel" "RelWithDebInfo")
 endif()
 
+SET(BOOST_ROOT ${CMAKE_CURRENT_SOURCE_DIR}/../boost)
+SET(BOOST_LIBRARYDIR ${BOOST_ROOT}/stage/lib)
+ADD_DEFINITIONS(-DBOOST_LIB_DIAGNOSTIC)
 if (MSVC) 
 # Boost
 find_package( Boost 1.54 COMPONENTS system thread log REQUIRED )
