diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1ece960..a6dca80 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -82,7 +82,7 @@ include_directories(
 find_package(Threads REQUIRED)
 
 # Boost
-find_package( Boost 1.55 COMPONENTS log thread system REQUIRED )
+find_package( Boost 1.55 COMPONENTS log thread system filesystem REQUIRED )
 include_directories( ${Boost_INCLUDE_DIR} )
 
 if(Boost_FOUND)
